pkgs/by-name/fi/fireplace/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkgs,
+}:
+stdenv.mkDerivation {
+  pname = "fireplace";
+  version = "unstable-2020-02-02";
+
+  buildInputs = [ pkgs.ncurses ];
+  nativeBuildInputs = [ pkgs.gcc ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install fireplace $out/bin/
+  '';
+
+  src = fetchFromGitHub {
+    owner = "Wyatt915";
+    repo = "fireplace";
+    rev = "aa2070b73be9fb177007fc967b066d88a37e3408";
+    hash = "sha256-2NUE/zaFoGwkZxgvVCYXxToiL23aVUFwFNlQzEq9GEc=";
+  };
+
+  meta = {
+    description = "A cozy fireplace in your terminal";
+    homepage = "https://github.com/Wyatt915/fireplace";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ multivac61 ];
```

Comment:
Please add yourself to the maintainers list, otherwise this is a failing evaluation.